pkg/controller/seed-controller-manager/mla/OWNERS

```diff
@@ -0,0 +1,13 @@
+# See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md
+
+approvers:
+  - sig-app-management
+
+reviewers:
+  - sig-app-management
+
+labels:
+  - sig/app-management
+
+options:
+  no_parent_owners: true
```

Comment:
I'm not sure do we want to keep this option or not. IMO, it makes sense to keep it, but it means that the SIG App Management would have to approve every change to the MLA controller.

Reply:
>SIG App Management would have to approve every change to the MLA controller.

Isn't that's the point?

Reply:
The main point is that they can approve their PRs themselves. This option means that we (SIG Cluster Management) wouldn't be able to approve changes to the MLA packages. It makes sense because the SIG App Management folks will mostly like want to be aware of changes to the MLA packages, but it might complicate the review process (although I don't think that's a problem).

Reply:
IIRC this option means if third-party folks make changes in MLA packages, they will need approvals from both sig cluster management and sig app management.
and for folks in sig app management, we will still need approval from sig cluster management
For example, this project-sync controller is owned by sig app: https://github.com/kubermatic/kubermatic/blob/master/pkg/controller/master-controller-manager/project-sync/OWNERS
but if you check this message: https://github.com/kubermatic/kubermatic/pull/6722#issuecomment-804154076
It still needs approval from sig cluster management

Reply:
so I think this option will require additional approval from sig-app, but will still require approval from sig-cluster

Reply:
@jiachengxu To my knowledge, that option is used to restrict folks defined in parent OWNERS file(s) to approve changes in that package. I searched for some docs on that option to confirm this, but I wasn't able to find anything.

Regarding #6722, I believe that SIG Cluster Management approval was required because it changed some files outside the `project-sync` package (such as `cmd/master-controller-manager/controllers.go`).

Reply:
@xmudrii You are right, thanks for explaining this to me :)